ios/FluentUI/Badge Field/BadgeViewTokenSet.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+/// Pre-defined styles of the Badge.
+@objc(MSFBadgeViewStyle)
+public enum Style: Int {
```

Comment:
oh i see how my comment is wrong. now we are publicly creaiting a "Style" enum that isn't necessarily associated with badgeview?

Reply:
maybe we should keep the badgeview extension here. curious to hear other teammates' thoughts

Reply:
This does not seem ideal - this defines a universal `Style` enum for all of Fluent, but then it's called `MSFBadgeViewStyle` in Objective-C. If it's truly meant to be BadgeView-specific then the naming needs to reflect that.

Reply:
Personally I'd vote to keep these enums inside `BadgeView` and keep the old namespaced lookup, though I notice that the equivalents for `Button` are defined as `ButtonStyle` and `ButtonSizeCategory`. @huwilkes, any thoughts?

Reply:
I think our general approach is to move things like styles and sizes in to the token files, and then have them named "ControlStyle" or "ControlSize" (now "ControlSizeCategory", but I think we've only updated button on that one) in Swift, then "MSFSwiftName" for Objective C. I think I'd actually prefer if we were better about having one thing per file, rather than putting a bunch of thinks in the token set file, but that's not really something we've done so far.